recipes/nodejs_from_binary.rb

```diff
@@ -57,5 +57,6 @@
   version node['nodejs']['version']
   checksum checksum
   has_binaries binaries
+  append_env_path node['nodejs']['binary']['append_env_path']
```

Comment:
Without this the node binary path was not added to the PATH env variable and the 'node' command was not found